taxi/views.py

```diff
@@ -14,14 +14,14 @@ def index(request):
     num_drivers = Driver.objects.count()
     num_cars = Car.objects.count()
     num_manufacturers = Manufacturer.objects.count()
-    num_visits = request.session.get("num_visits", 0)
+    num_visits = request.session.get("num_visits", 1)
     request.session["num_visits"] = num_visits + 1
 
     context = {
         "num_drivers": num_drivers,
         "num_cars": num_cars,
         "num_manufacturers": num_manufacturers,
-        "num_visits": num_visits + 1,
+        "num_visits": num_visits,
     }
```

Comment:
how do you think, num_visits contains number or reference?
Because let's imagine that request.session.num_visits is 5
Then
```
num_visits = 5
request.session.num_visits = 6
then what is num_visits? 5 or 6? Does it contain reference to some object and number will be automatically updated to 6 or it just contains a number that won't be updated?
```